### PR TITLE
`username` instead of `name` for user UUID

### PR DESCRIPTION
**Pull request by jessieay**
* `name` is the UUID for a user record
* Reference: https://github.com/18F/team-api/issues/113

### CODE REVIEW

File: test/api_test.rb [hunk 1]
```diff
@@ -223,7 +223,7 @@ def parse_page_content(page_url)
     end
 
     def user_snippet_from_batch(datestamp, username)
-      snippets[datestamp].detect { |snippet| snippet['name'] == username }
+      snippets[datestamp].detect { |snippet| snippet['username'] == username }
```
Comment (jessieay) [inline test/api_test.rb]: I'm not 100% sure this change is correct -- but specs are all passing ?

**Comment by jessieay** [comment]
cc @mbland

**Comment by mbland** [comment]
`./go test` fails, and `./go lint` also catches a lot of line length errors.

**Comment by jessieay** [comment]
ah oK, will add that info to the README

**Comment by jessieay** [comment]
going to close this for now. seems like there are more pressing API needs and we might want to standardize on a different field name (such as `ID`) anyway

**Pull request closed without merging**